packages/astro-imagetools/api/utils/getImgElement.js

```diff
@@ -62,7 +62,7 @@ export default function getImgElement({
   const imgElement = `<img
     ${attributesString}
     src="${src}"
-    ${typeof alt === "string" ? `alt="${alt}"` : ""}
+    ${typeof alt === "string" ? `alt="${alt}"` : `alt=""`}
```

Comment:
This might be a little simpler, rather than nested backticks.

```suggestion
    alt="${typeof alt === 'string' ? alt : ''}"
```

Reply:
YES! I was aiming for this as well, but decided to keep it close to what it was.
Thanks for your suggestion I apply it. 👍